app/experimenter/nimbus-ui/src/components/RichText/index.test.tsx

```diff
@@ -13,10 +13,10 @@ it("renders text with newlines converted to <br> tags", () => {
 
   We are not ready`;
 
-  const result = render(<RichText {...{ text }} />);
+  const view = render(<RichText {...{ text }} />);
```

Comment:
Huh, we used `result` because the type is called `RenderResult`. We could also probably destructure `container` from here instead. Seems like [an arbitrary rule](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/render-result-naming-convention.md), but 🤷‍♂️ I like a robot keeping the code clean